buildroot/share/scripts/upload.py

```diff
@@ -124,6 +129,7 @@ def _RemoveFirmwareFile(FirmwareFile):
     marlin_board_info_name = _GetMarlinEnv(MarlinEnv, 'BOARD_INFO_NAME')
     marlin_board_custom_build_flags = _GetMarlinEnv(MarlinEnv, 'BOARD_CUSTOM_BUILD_FLAGS')
     marlin_firmware_bin = _GetMarlinEnv(MarlinEnv, 'FIRMWARE_BIN')
+    marlin_long_filename_host_support = _GetMarlinEnv(MarlinEnv, 'LONG_FILENAME_HOST_SUPPORT') is not None
```

Comment:
This configuration option specifically signifies the presence of `M33`.

Reply:
It is also used in M20 to retrieve the LFN list with 'L' flag

Reply:
Hmm, so it does.